utils/k8s/openebscrd.go

```diff
@@ -40,3 +40,22 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
 	spcClient := k8s.OpenebsClientSet.Openebs().StoragePoolClaims()
 	return spcClient.Delete(spcName, &meta_v1.DeleteOptions{})
 }
+
+// GetCStorPool returns the CStorPool object for given cStorPoolName.
+// :return: *openebs_v1.CStorPool: Pointer to CStorPool objects.
```

Comment:
@aswathkk 
We might not require this line(45):

Reply:
@sonasingh46
Why?